<!-- -->
<launch>
    <!-- 先开启障碍物检测 -->
    <include file="$(find lidar_detection)/launch/Yunlecar_lidar_detection.launch" />

    <arg name="resolution" default="0.5" />
    <arg name="cell_width" default="50" />
    <arg name="cell_height" default="50" />
    <arg name="points_topic" default="/ray_filter/velodyne_points_costmap" />
    <arg name="offset_x" default="12.5" />
    <arg name="offset_y" default="0.0" />
    <arg name="offset_z" default="-2.0" />

	<node pkg="points2octomap" type="points2octomap" name="points2octomap" output="screen">
        <param name="resolution" value="$(arg resolution)" />
        <param name="cell_width" value="$(arg cell_width)" />
        <param name="cell_height" value="$(arg cell_height)" />
        <param name="points_topic" value="$(arg points_topic)" />
        <param name="offset_x" value="$(arg offset_x)" />
        <param name="offset_y" value="$(arg offset_y)" />
        <param name="offset_z" value="$(arg offset_z)" />
	</node>
</launch>